ui/js/components/auth.js

```diff
@@ -79,6 +79,10 @@ treeherder.component("login", {
                             // is still set to the logged in user.  So we just
                             // double-check the localStorage value and possibly
                             // skip the logout.
+                            //
+                            // This is probably not needed now due to
+                            // fix for bug 1339659, but no harm in leaving it
+                            // for now.
```

Comment:
I'm not 100% sure we can. I'm happy to remove it for now and we can add it back later if it turns out that the hack was there to fix another issue instead.